common/pb/serverinfo_ban.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto2";
+message ServerInfo_Ban {
+    optional string admin_id = 1;
+    optional string admin_name = 2;
```

Comment:
I think admin_id and name should be required; all bans have an admin who placed them.